cmd/controller.go

```diff
@@ -100,17 +101,46 @@ func parseControllerConfig() {
 			log.Error(err, "Can't get k8s client")
 			os.Exit(1)
 		}
+
 		CSINodeDsName := "juicefs-csi-node"
 		if name := os.Getenv("JUICEFS_CSI_NODE_DS_NAME"); name != "" {
 			CSINodeDsName = name
 		}
-		ds, err := k8sclient.GetDaemonSet(context.TODO(), CSINodeDsName, config.Namespace)
+		labelSelector := &metav1.LabelSelector{
+			MatchLabels: map[string]string{
+				"app": "juicefs-csi-node",
+			},
+		}
+
+		pods, err := k8sclient.ListPod(context.TODO(), config.Namespace, labelSelector, nil)
```

Comment:
add a new func, like ListPodWithLimit， and `listOptions.limit`=1?

Reply:
🤔 hmm, this action of getting the csi node pod only happens once when the csi controller starts, is it necessary to add a new func? 
However, perhaps there will be more scenarios where this method `ListPodWithLimit ` will be used in the future